### PR TITLE
Add new fbs purchase order creation endpoint documentation

### PR DESCRIPTION
**Pull request by Psilolouben**
Addresses [this task](https://phabricator.skroutz.gr/T118679)

### CODE REVIEW

File: source/localizable/smart_cart/fbs_api.html.md.erb [hunk 1]
```diff
@@ -72,6 +73,24 @@ curl -X GET https://<%= settings.api_domain %>/merchants/ecommerce/fbs/purchase_
...
+#### Example
+<pre class="terminal">
+curl -X POST https://<%= settings.api_domain %>/merchants/ecommerce/fbs/purchase_orders \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Authorization: Bearer your_access_token_here'
+  -d '{ "expected_date": "2022-09-22", "line_items": [ { "product_uid": "52781", "expected_quantity": 8, "keep_serial_numbers": true, "has_expiration_date": false, "storage_conditions": "2" } ] }'
+</pre>
+
+<%= render_recording :merchants_fbs_purchase_orders_create %>
+
```
Comment (kyrpas) [inline source/localizable/smart_cart/fbs_api.html.md.erb:93]: How about adding a link to the appendix here? It took me a little bit to realise there is another section below explaining some fields in more detail (like storage_conditions).

Something like: `View purchase order creation options in more detail`

**Review by kyrpas — CHANGES_REQUESTED** [review]
Looks good. 👍  
Couple of minor comments.

File: source/localizable/smart_cart/_fbs_purchase_order_object.html.md.erb [hunk 2]
```diff
@@ -17,3 +17,23 @@ Name                              | Type   | Description
...
+`new_purchase_order.expected_date`    | Date String in format<br> `YYYY-MM-DD` | "2022-09-22" | Date the Purchase Order is expected to reach the Skroutz warehouses
+`new_purchase_order.line_items`       | Array  |             | The Purchase Order line items
+
+#### Purchase Order creation options: Line items
+
+Name                              | Type   | Example     | Description
+--------------------------------- | ------ | ----------- | --------------------------------------------------
+`line_item.product_uid`           | String | "52781"     | Unique product id
+`line_item.expected_quantity`     | Integer|       8     | The quantity of this product that the Skroutz warehouse will receive
+`line_item.keep_serial_numbers`   | Boolean|     true    | Product requires serial number_of_parcels
```
[the remark this answers is not in the document]
Comment (Psilolouben) [reply source/localizable/smart_cart/_fbs_purchase_order_object.html.md.erb]: [fixed](https://github.com/skroutz/developer.skroutz.gr/pull/192/commits/f2bf5b5feb7f4033ca86af30dbce328a2e946b83)

File: source/localizable/smart_cart/fbs_api.html.md.erb (hunk 1 above)
Comment (Psilolouben) [reply source/localizable/smart_cart/fbs_api.html.md.erb:93]: [fixed](https://github.com/skroutz/developer.skroutz.gr/pull/192/commits/40680b2c86bcf8f2607cbf2a556ca06abb27b7e9)

**Pull request merged**